src/main/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/alignment/AssemblyContigAlignmentsConfigPicker.java

```diff
@@ -434,11 +439,12 @@ private static AssemblyContigWithFineTunedAlignments updateContigMappingsWithGap
 
     /**
      * when two configurations are the same,
-     * put the one with less alignments,
-     * or less summed mismatches if still tie
-     * first
+     * implement ordering that
+     *      prefer the configuration with less alignments, then
+     *      prefer the configuration less summed mismatches if still tie
```

Comment:
"and prefers the configuration with a lower number of summed mismatches in case of a tie"

Reply:
done